src/DataGrid.tsx

```diff
@@ -548,6 +548,12 @@ function DataGrid<R, SR, K extends Key>(
     if (!isCellEvent && !isRowEvent) return;
 
     const { keyCode } = event;
+    const cKey = 67;
+    const vKey = 86;
+
+    if (isCtrlKeyHeldDown(event) && keyCode === cKey && window.getSelection()?.toString() !== '') {
```

Comment:
Can/should we check that the selection starts/ends in the cell?
https://developer.mozilla.org/en-US/docs/Web/API/Selection#instance_properties
I wonder if we can do something like `cell.contains(selection.anchorNode)` for example.

How should we handle the `isRowEvent` case?

Reply:
> Can/should we check that the selection starts/ends in the cell?

Why do we need to check this? When the text selection is across multiple cells, it doesn't open the editor and performs the browser behaviour.
![Recording 2024-01-18 at 10 37 19](https://github.com/adazzle/react-data-grid/assets/54691695/6f6c6d49-47ff-4b66-b4b6-16f32f17afe5)

Reply:
I can click on a cell, select text elsewhere without losing focus on the cell, then press ctrl+c, what should happen then? Right now we open the selected cell and copy the selected cell's text.

![Recording 2024-01-18 at 11 19 14](https://github.com/adazzle/react-data-grid/assets/567105/a512065a-851d-46b8-9f48-1972e957cf6b)

I can also select text in a cell, tab to the next cell, and the selection will remain on the previous cell. Same issue as above.

![Recording 2024-01-18 at 11 17 41](https://github.com/adazzle/react-data-grid/assets/567105/d1766c95-9ef1-4135-959b-52a5de17e423)

@amanmahajan7 any ideas?

Reply:
For both scenarios, it doesn't open the editor and copies the selected text, is it not desirable?
![Recording 2024-01-18 at 11 37 36](https://github.com/adazzle/react-data-grid/assets/54691695/3b8913ae-c1b0-4e4f-a5f4-f7f4952fec35)

Reply:
Mmh, yes that seems correct, maybe I'm confusing myself.

Reply:
We don't have special handling for `isRowEvent` and it copies text as expected, is there anything we need to consider?